nan.lisp

```diff
@@ -0,0 +1,41 @@
+
+(in-package #:org.shirakumo.float-features)
+
+;; let's forget about long/short
+
+(handler-case
+    (progn
+      (bits-short-float 0)
+      (defconstant short-float-nan
+        (bits-short-float #b0111111000000000)))
+  (error ()
+    (define-symbol-macro short-float-nan
+        (bits-short-float #b0111111000000000))))
```

Comment:
I think these definitions could all be shortened to just
```
(define-symbol-macro short-float-nan (load-time-value (bits-short-float ...)))
```
Or in the very least could use a macrolet to avoid the repetition.

Reply:
load-time value does not constant propagate.

Reply:
Ah, that's a good point.